README.md

```diff
@@ -31,8 +42,24 @@ as provided by [Frab][frab-github]. The file format produced by the predecessor
 * In general it is possible to re-deployed the app for other events which
 provide the same event file format.
 
+## Build instructions
+
+Build instructions for configuring your build variants (i.e. debug and release).
+
+### Debug build / signing
+
+Before importing the project in Android Studio
+
+1. Create a file `gradle.properties` under directory `app`.
+2. Add the following JVM arguments in the file file `app/gradle.properties`.
+    ```
+    # Specifies the JVM arguments used for the daemon process.
+    # The setting is particularly useful for tweaking memory settings.
+    org.gradle.jvmargs=-Xmx2048m -Xms512m -XX:MaxPermSize=768m
```

Comment:
I would prefer to set good defaults once for both build types and not put the burden onto every user. At the moment I don't have gradle settings configured for this project because I have a configuration file in `~/.gradle/gradle.properties` which applies to all projects of the logged-in user. I would prefer to exclude this aspect from this pull request to first discuss a suitable solution in a separate issue. If you agree please feel free to create a new issue.

Reply:
Okay. I removed the JVM arguments. I updated it in this commit 
74c1219

Reply:
I created a new issue #146 to discuss regarding the jvm arguments.
I excluded them from this PR.